mock/py/mockbuild/config.py

```diff
@@ -247,7 +248,6 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
     config_opts['target_arch'] = 'i386'
     config_opts['releasever'] = None
     config_opts['rpmbuild_arch'] = None  # <-- None means set automatically from target_arch
-    config_opts['dnf_vars'] = {}
```

Comment:
When you remove this, users will have to do this in the configuration file anyway which is inconvenient.